be/src/common/config.h

```diff
@@ -286,6 +286,9 @@ CONF_mInt64(min_compaction_failure_interval_sec, "600"); // 10 min
 CONF_mInt32(min_compaction_threads, "10");
 CONF_mInt32(max_compaction_threads, "10");
 
+// This config can be set to limit thread number in tablet meta checkpoint thread pool.
```

Comment:
```suggestion
// Thread count to do tablet meta checkpoint, -1 means use the data directories count.
```
Or some words like this.

Reply:
done